pwa/src/components/tag/Tag.tsx

```diff
@@ -21,8 +21,8 @@ interface TagIconLinkProps {
   href: string;
 }
 
-export const TagIconLink: React.FC<TagIconLinkProps> = ({ icon, href, tag }) => (
-  <div className={clsx(styles.tag, styles.tagIconLink)} onClick={() => navigate(href)}>
+export const TagIconLinkExternal: React.FC<TagIconLinkProps> = ({ icon, href, tag }) => (
+  <div className={clsx(styles.tag, styles.tagIconLink)} onClick={() => open(href)}>
     {tag && <span>{tag}</span>}
     <span>{icon}</span>
   </div>
```

Comment:
We're duplicating a lot of code this way (I understand why you did it though).

I think another possible solution would be to have the following props:

```
interface TagProps {
  content: JSX.Element; // icon? label? doesn't matter anymore, let the implementation decide
  onClick: () => any; // external? internal? doesn't matter anymore, let the implementation decide
}
```